hash/megacd.xml

```diff
@@ -136,6 +135,18 @@ Dumps that are known to exist, but are lost in the aether:
 | Virtual VCR - Colors of Modern Rock (Prototype)             |                                                                   |
 +=============================================================+===================================================================+
 
+Limited Run reissues:
++=============================================================+===================================================================+
+|                            Title                            |                               Notes                               |
++=============================================================+===================================================================+
+| The Secret of Monkey Island                                 | Reissued by Limited Run in 2020. Only 2500 copies were available. |
+|                                                             | CRCs differ from original release but data should be identical.   |
++=============================================================+===================================================================+
+| Star Wars: Rebel Assault                                    | Reissued by Limited Run in 2020. Only 2500 copies were available. |
+|                                                             | Data track is identical to original release, but track 2 has a    |
+|                                                             | different CRC. Gameplay should be unchanged.                      |
++=============================================================+===================================================================+
```

Comment:
Are these placeholders for undumped discs, or is this just trivia?  If it’s just trivia, it feels like stuff that doesn’t belong here.

Reply:
I figured both were still too new to be added (less than five years since release, despite them being OOP) but were worth mentioning. They have been dumped for a while.

If you want this info removed, then no problem.

Reply:
The general rule is three years since release.  These are re-issues of old games anyway, so I’d even be willing to bend the three-year rule a bit.